src/languages/en.ts

```diff
@@ -2204,6 +2204,37 @@ export default {
             subsidiarySelectDescription: "Choose the subsidiary in NetSuite that you'd like to import data from.",
             noSubsidiariesFound: 'No subsidiaries found',
             noSubsidiariesFoundDescription: 'Add the subsidiary in NetSuite and sync the connection again.',
+            tokenInput: {
+                title: 'NetSuite setup',
+                formSteps: {
+                    installBundle: {
+                        title: 'Install the Expensify bundle',
+                        description: 'In NetSuite, go to *Customization > SuiteBundler > Search & Install Bundles* > search for "Expensify" > install the bundle.',
+                    },
+                    enableTokenAuthentication: {
+                        title: 'Enable token-based authentication',
+                        description: 'In NetSuite, go to *Setup > Company > Enable Features > SuiteCloud* > enable *token-based authentication*.',
```

Comment:
@jamesdeanexpensify @yuwenmemon the copy I used from the doc @jamesdeanexpensify shared [here](https://github.com/Expensify/App/issues/43434#issuecomment-2197329774). In the design doc, `token-based` is `Token-case`.  Would like to confirm once as this could be on NetSuite.

Reply:
What you have now is accurate :+1: